scripts/deployHyperProverAndInbox.ts

```diff
@@ -0,0 +1,142 @@
+import { ethers, run, network } from 'hardhat'
```

Comment:
we should do some script cleanups soon - this is one of mine and its p superfluous at this point